pyaedt/application/Analysis.py

```diff
@@ -1569,7 +1570,8 @@ def analyze_setup(self, name, num_cores=None, num_tasks=None, num_gpu=None, acf_
                 return False
         if set_custom_dso:
             self.set_registry_key(r"Desktop/ActiveDSOConfigurations/" + self.design_type, active_config)
-        self.logger.info("Design setup %s solved correctly", name)
+        end = (time.time() - start) / 60
+        self.logger.info("Design setup %s solved correctly in %s min", name, end)
```

Comment:
```suggestion
        m, s = divmod(time.time() - start, 60)
        h, m = divmod(m, 60)
        self.logger.info("Design setup {} solved correctly in {:d}h {:02d}m {:02d}s".format(name, h, m, s))
```
This looks a bit better for simulations that last few seconds and for longer simulations.
It works in Ironpython and it does not require additional import.